tools/tpm2_verifysignature.c

```diff
@@ -321,7 +322,7 @@ int execute_tool(int argc, char *argv[], char *envp[], common_opts_t *opts,
 
     tpm2_verifysig_ctx ctx = {
             .flags = { .all = 0 },
-            .halg = TPM_ALG_SHA256,
+            .halg = TPM_ALG_SHA1,
             .msgHash = TPM2B_TYPE_INIT(TPM2B_DIGEST, buffer),
```

Comment:
When I update the man page for it, the default is mentioned in the common file, so if I split it into a separate commit, there's a point in time where the man page doesn't match reality, which I don't think we would ever want to happen. Splitting it doesn't buy us anything, because if we revert it, we'd have to revert the other patch as well.